ThemingCodelab/app/src/main/java/com/codelab/theming/ui/start/Home.kt

```diff
@@ -79,7 +86,11 @@ fun Home() {
 private fun AppBar() {
     TopAppBar(
         navigationIcon = {
-            Icon(Icons.Rounded.Palette, Modifier.padding(horizontal = 12.dp))
+            Icon(
+                imageVector = Icons.Rounded.Palette,
+                contentDescription = stringResource(R.string.label_app_icon),
```

Comment:
This icon is currently used only for decorative purposes and does not represent a meaningful action that a user can take. Therefore you want to set its contentDescription to `null`.